kiwi/defaults.py

```diff
@@ -61,6 +61,12 @@ def __init__(self):
         ]
 
     @classmethod
+    def is_obs_worker(self):
+        # the presence of /.buildenv on the build host indicates
+        # we are building inside of the open buildservice
+        return os.path.exists('/.buildenv')
+
```

Comment:
Sounds reasonable to me. I don't really know how obs:// is converted to suse://, for some reason I though it was done by OBS before running kiwi, so that I considered suse:// to be our preferred indicator.

I agree the runtime checks should be as early as possible and centralized as possible.